.eslintrc

```diff
@@ -82,10 +82,6 @@
         "@typescript-eslint/no-unused-vars": [
           "warn",
           { "varsIgnorePattern": "^_" }
-        ],
```

Comment:
metabase-lib heavily depends on this

Reply:
Would be nice to enforce `@typescript-eslint/no-use-before-define` it again at some point. Let's maybe have an issue for this?

Reply:
I think the main idea is that we need to add proper types to metabase-lib, cleanup code a bit, and remove ts-nocheck. I'll outline in the RFC